app/src/pages/inside/common/statusDropdown/statusDropdown.jsx

```diff
@@ -120,17 +132,25 @@ export class StatusDropdown extends Component {
     const { intl } = this.props;
 
     const STATUS_TYPES = status === IN_PROGRESS ? [] : [PASSED, FAILED, SKIPPED];
+    const isTestItemsPage = this.getIsExpectedCurrentPage(TEST_ITEM_PAGE);
 
     if (STATUS_TYPES.indexOf(status) < 0) STATUS_TYPES.push(status);
 
     return STATUS_TYPES.map((item) => ({
-      label: formatStatus(intl.formatMessage, item),
+      label: isTestItemsPage ? (
```

Comment:
Use more abstract and generic condition here, f.e. `withIndicator`.
Or, may be it will be better, can we use just one status label component and manage its content via the prop?